docs/src/univariate.md

```diff
@@ -71,7 +71,7 @@ cf(::UnivariateDistribution, ::Any)
 insupport(::UnivariateDistribution, x::Any)
 pdf(::UnivariateDistribution, ::Real)
 logpdf(::UnivariateDistribution, ::Real)
-loglikelihood(::UnivariateDistribution, ::AbstractArray)
+loglikelihood(::UnivariateDistribution, ::Union{Real,AbstractArray})
```

Comment:
not a fan of using the two, since we should be able to broadcast `loglikelihood.(d, xs)` but I can see the consistency with rand

Reply:
What arguments `d` and `xs` do you have in mind? E.g., `loglikelihood.(d::UnivariateDistribution, xs::AbstractArray{<:Real})` only works with this PR (though I'm not sure what its use case would be) and `loglikelihood.(d::UnivariateDistribution, xs::AbstractArray{<:AbstractArray{<:Real}})` is not affected by this PR.

Reply:
oh ok my bad, I thought `loglikelihood.(d::UnivariateDistribution, xs::AbstractArray{<:Real})` already worked and this PR added `loglikelihood(d::UnivariateDistribution, xs::AbstractArray{<:Real})` without broadcast with the same behaviour